### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by cody82** [hardbyte/python-can#1544] [General]
There are some important fixes for me since 4.1.0. When is a new release planned?

**Comment by zariiii9003** [comment]
Soonish #1552

**Comment by zariiii9003** [comment]
[4.2.0rc0](https://github.com/hardbyte/python-can/releases/tag/v4.2.0rc0)

**Comment by hardbyte** [comment]
Nowish https://github.com/hardbyte/python-can/releases/tag/v4.2.0
